server/container/guice/cassandra-rabbitmq-guice/pom.xml

```diff
@@ -110,10 +115,6 @@
             <artifactId>event-sourcing-event-store-memory</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>${james.groupId}</groupId>
```

Comment:
If I understand correctly `jame-server-cassandra-guice` is in fact `jame-server-cassandra-activemq-guice` ?

Reply:
Yes. A guice server bundling Cassandra, ES, ActiveMQ.

We suffer from having no clearer separation between our guice modules and our apps.

I would be in favor of something like this:

```
server/apps/spring
server/apps/memory
server/apps/distributed
server/apps/jpa
server/apps/jpa-smtp
server/apps/cassandra //-es-activemq
server/containers/guice/... // all module definitions would go here
```